### ISSUE TITLE
Sticky note text not wrapping on image exports

### ISSUE THREAD

**Issue by tacomanator** [tldraw/tldraw#925]
Create a sticky note with multiple lines of text, then export as PNG, JPG, or WEBP. On the screen, the text wraps within the sticky note. On the exported files the text does not wrap and overflows the sticky note on the left side. On Safari, the text is missing completely.

### Screenshot

<img width="784" alt="Screen Shot 0004-08-22 at 8 30 56" src="https://user-images.githubusercontent.com/63794/185815495-31d13671-6ccd-4493-89bc-16e2b887722c.png">

### Export (PNG) from Firefox 102 (also reproduced in Chrome 104):

![export](https://user-images.githubusercontent.com/63794/185815502-2838260f-7502-418e-99cc-67b663dc50dc.png)

### Export (PNG) from Safari:

![Page 2-2](https://user-images.githubusercontent.com/63794/185815670-afebb1d7-44da-4a1f-bc6c-946953b9eb9f.png)

### File:

```json
{
  "name": "New Document",
  "fileHandle": null,
  "document": {
    "id": "doc",
    "name": "New Document",
    "version": 15.5,
    "pages": {
      "page": {
        "id": "page",
        "name": "Page 2",
        "childIndex": 1,
        "shapes": {
          "5b2fffcd-c804-427c-1ad8-bced2f54a454": {
            "id": "5b2fffcd-c804-427c-1ad8-bced2f54a454",
            "type": "sticky",
            "name": "Sticky",
            "parentId": "page",
            "childIndex": 4,
            "point": [
              179.36,
              -187.54
            ],
            "size": [
              200,
              200
            ],
            "text": " 即時エラーの場合、クライエントにエラーを返して、クライエント側で対応する",
            "rotation": 0,
            "style": {
              "color": "black",
              "size": "small",
              "isFilled": false,
              "dash": "draw",
              "scale": 1,
              "font": "script",
              "textAlign": "middle"
            }
          },
          "451afdb9-95a6-459c-3416-78f30cae5832": {
            "id": "451afdb9-95a6-459c-3416-78f30cae5832",
            "type": "sticky",
            "name": "Sticky",
            "parentId": "page",
            "childIndex": 5,
            "point": [
              176.55,
              39.65
            ],
            "size": [
              200,
              200
            ],
            "text": "Create a sticky note with multiple lines of text, then export as PNG, JPG, or WEBP. ",
            "rotation": 0,
            "style": {
              "color": "black",
              "size": "small",
              "isFilled": false,
              "dash": "draw",
              "scale": 1,
              "font": "script",
              "textAlign": "middle"
            }
          },
          "3132ab41-0c8a-41f1-353f-8c00fe62eb3c": {
            "id": "3132ab41-0c8a-41f1-353f-8c00fe62eb3c",
            "type": "rectangle",
            "name": "Rectangle",
            "parentId": "page",
            "childIndex": 6,
            "point": [
              -179.11,
              -225.86
            ],
            "size": [
              688.16,
              507.52
            ],
            "rotation": 0,
            "style": {
              "color": "black",
              "size": "small",
              "isFilled": false,
              "dash": "draw",
              "scale": 1
            },
            "label": "",
            "labelPoint": [
              0.5,
              0.5
            ]
          }
        },
        "bindings": {}
      }
    },
    "pageStates": {
      "page": {
        "id": "page",
        "selectedIds": [],
        "camera": {
          "point": [
            367.16,
            329.17
          ],
          "zoom": 1.0739975378685118
        },
        "editingId": null
      }
    },
    "assets": {}
  },
  "assets": {}
}
```

**Comment by steveruizok** [comment]
Working on a fix here! https://github.com/tldraw/tldraw/pull/936

**Comment by steveruizok** [comment]
Fixed by #936!

![image](https://user-images.githubusercontent.com/23072548/187060296-e85ba0c7-abaf-46e5-96d8-2bc1e154fb39.png)
